pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  flutter_lints: any
+  flutter_lints: ^3.0.1
```

Comment:
We can still use `any`?

Reply:
Because the project did not upload the pubspec.lock file, the local usage deps and the remote CI were no same, so CI often failed to pass of the lints.
I recommend using the specified version, does it have any other effects?

Reply:
Because the package resolves different Dart SDK between its major versions:
- 3.x: >=3.0.0
- 2.x: >=2.17.0
- 1.x: >=2.12.0

Since our package supports `>=2.13.0`, using `any` will help to develop the package with different SDK versions.

Reply:
As far as I know, reference relationships of dev_dependencies are not transitive.
https://dart.dev/tools/pub/dependencies#dev-dependencies

Reply:
However, changing it back is not a big problem. If you encounter CI problems in the future, just run `dart fix`.